instance/migrations/0102_auto_20180419_1648.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2018-04-19 16:48
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import instance.models.mixins.database
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0101_auto_20180418_1601'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='mongodb_server',
+            field=models.ForeignKey(blank=True, default=instance.models.mixins.database.select_random_mongodb_server, null=True, on_delete=django.db.models.deletion.PROTECT, to='instance.MongoDBServer'),
```

Comment:
@rocioar Looks like the difference between this and 0100 is this default - can't it be done in 0100?

Another question - should this field remain nullable at the end? It's a geniune question, not a suggestion to change something :)

Reply:
I updated it so that it's all done on the same migration file.

About this field being nullable, yes, I think so, it's going to be null when a replica set is setup instead of a single server.